src/behavior-considered-undefined.md

```diff
@@ -62,9 +61,9 @@ A reference/pointer is "dangling" if it is null or not all of the bytes it
 points to are part of the same allocation (so in particular they all have to be
 part of *some* allocation). The span of bytes it points to is determined by the
 pointer value and the size of the pointee type. As a consequence, if the span is
-empty, "dangling" is the same as "non-null". Note that slices point to their
+empty, "dangling" is the same as "non-null". Note that slices/strings point to their
```

Comment:
```suggestion
empty, "dangling" is the same as "non-null". Note that slices and strings point to their
```
(https://github.com/rust-lang/reference/blob/master/STYLE.md#language-and-grammar)